### PR TITLE
Update Get Started Guide with instruction of HWLOC usage

### PR DESCRIPTION
**Pull request by alexandraepan**
### Description 
Update Get Started Guide with instruction of HWLOC usage.

Fixes # - _issue number(s) if exists_

- [x] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [x] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [x] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [x] not needed

### Breaks backward compatibility
- [ ] Yes
- [x] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### Other information
Signed-off-by: Alexandra Epanchinzeva alexandra.epanchinzeva@intel.com

### CODE REVIEW

File: doc/GSG/hybrid_cpu_support.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.. _hybrid_cpu_support:
+
+Hybrid CPU and NUMA Support
+***************************
+
+If you need NUMA/Hybrid CPU support in oneTBB, you need to make sure that HWLOC* is installed on your system.
+
+HWLOC* (Hardware Locality) is a library that provides a portable abstraction of NUMA and hybrid CPU systems.
```
Comment (isaevil) [inline doc/GSG/hybrid_cpu_support.rst:8]: Can we reuse a definition from the official website?
```suggestion
HWLOC* (Hardware Locality) is a library that provides a portable abstraction of the hierarchical topology of modern architectures (NUMA, hybrid CPU systems, etc). 
```
What do you think?

File: doc/GSG/hybrid_cpu_support.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Install HWLOC*
+^^^^^^^^^^^^^^
+
+To install HWLOC*, visit the official Portable Hardware Locality website (https://www-lb.open-mpi.org/projects/hwloc/).
+
+* For Windows* OS, binaries are available for download. 
+* For Linux* OS, only the source code is provided and binaries should be built. 
+
+On Linux* OS, HWLOC* can be also installed with packet managers, such as APT*, YUM, etc.
```
[the remark this answers is not in the document]
Comment (alexandraepan) [reply doc/GSG/hybrid_cpu_support.rst]: Good catch, a typo indeed :)

File: doc/GSG/hybrid_cpu_support.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+To check if HWLOC* is already installed on your system, run `hwloc-ls`:
+
+   * For Linux* OS, in the command line. 
+   * For Windows* OS,  in the command prompt. 
+
+If HWLOC* is installed, the command displays information about the hardware topology of your system. 
+If it is not installed, you receive an error message saying that the command ``hwloc-ls`` could not be found.
+
+.. note:: Make sure that HWLOC* is version 2.5 or higher.
```
[the remark this answers is not in the document]
Comment (alexandraepan) [reply doc/GSG/hybrid_cpu_support.rst]: @isaevil Thanks,! updated

File: doc/GSG/hybrid_cpu_support.rst (hunk 1 above)
Comment (alexandraepan) [reply doc/GSG/hybrid_cpu_support.rst]: Updated.

File: doc/GSG/hybrid_cpu_support.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Install HWLOC*
+^^^^^^^^^^^^^^
+
+To install HWLOC*, visit the official Portable Hardware Locality website (https://www-lb.open-mpi.org/projects/hwloc/).
+
+* For Windows* OS, binaries are available for download. 
+* For Linux* OS, only the source code is provided and binaries should be built. 
+
+On Linux* OS, HWLOC* can be also installed with packet managers, such as APT*, YUM*, etc.
```
Comment (omalyshe) [inline doc/GSG/hybrid_cpu_support.rst:35]: packet -> package

Comment (alexandraepan) [reply doc/GSG/hybrid_cpu_support.rst]: thanks, have updated

**Review by omalyshe — APPROVED** [review]

**Pull request merged**